client/landing/stepper/declarative-flow/internals/steps-repository/domain-transfer-domains/use-validation-message.ts

```diff
@@ -83,6 +83,12 @@ export function useValidationMessage( domain: string, auth: string, hasDuplicate
 			saleCost: validationResult.sale_cost,
 			currencyCode: validationResult.currency_code,
 		};
+	} else if ( validationResult?.is_price_limit_exceeded === true ) {
+		return {
+			valid: false,
+			loading: false,
+			message: __( "Sorry, we still don't support some higher tier premium domain transfers." ),
```

Comment:
```suggestion
			message: __( "Sorry, we don't support some higher tier premium domain transfers." ),
```
What do you think of this suggestion?